e2e/config/config.go

```diff
@@ -45,9 +58,10 @@ type Cluster struct {
 
 type Config struct {
 	// User configurable values.
-	Repo     Repo
-	Clusters map[string]Cluster
-	PVCSpecs []PVCSpec
+	Repo         Repo
+	DRNamespaces DRNamespaces
```

Comment:
So you want the config to be:

```yaml
drNamespaces:
  ramenHubNamespace: ramen-system
```

I think "namespaces" is better, matches better other names (clusters, tests, pvcspecs). Everything in this config is in the context of dr anyway.